lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -307,7 +307,7 @@ def lookup_svp(self, temperature):
                 A cube of saturated vapour pressures (Pa).
         """
         T_min = svp_table.T_MIN
-        T_max = svp_table.T_MAX
+        T_max = svp_table.T_MAX - svp_table.T_INCREMENT
```

Comment:
You could put a comment here to say what this is doing. What do you think?

Reply:
Done.